packages/aws-cdk-lib/aws-eks/README.md

```diff
@@ -207,6 +206,24 @@ cluster.addNodegroupCapacity('custom-node-group', {
 });
 ```
 
+To define the type of the AMI for the node group, you may explicitly define `amiType` according to your requirements, supported amiType could be found [HERE](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_eks.NodegroupAmiType.html).
+
+```ts
+declare const cluster: eks.Cluster;
+
+// X86_64 based AMI managed node group
+cluster.addNodegroupCapacity('custom-node-group', {
+  instanceTypes: [new ec2.InstanceType('m5.large')], // NOTE: if amiType is x86_64-based image, the instance types here must be x86_64-based.
+  amiType: eks.NodegroupAmiType.AL2023_X86_64_STANDARD,
+});
+
+// ARM_64 based AMI managed node group
+cluster.addNodegroupCapacity('custom-node-group', {
+  instanceTypes: [new ec2.InstanceType('m6g.medium')], // NOTE: if amiType is ARM-based image, the instance types here must be ARM-based.
+  amiType: eks.NodegroupAmiType.AL2023_ARM_64_STANDARD,
+});
+```
```

Comment:
Discussion: should we address AL2023 here? or maybe keep it as default `AL2_ARM_64`.